### PR TITLE
[MM-676]: Fixed issue of meeting link being posted when user successfully connects to MS Teams Meetings

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
### Summary
Fixed the issue of the meeting link being posted when a user successfully connects to MS Teams Meetings

#### Existing
image
#### Fixed
image

### CODE REVIEW

File: server/http.go [hunk 1]
```diff
@@ -162,7 +162,7 @@ func (p *Plugin) completeUserOAuth(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	user, appErr := p.API.GetUser(userID)
+	_, appErr := p.API.GetUser(userID)
 	if appErr != nil {
 		p.API.LogError("complete oauth, error getting MM user", "error", appErr.Error())
 		http.Error(w, appErr.Error(), http.StatusInternalServerError)
```
Comment (raghavaggarwal2308) [inline server/http.go]: Can we remove this function call now?

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]

File: server/http.go (hunk 1 above)
Comment (Kshitij-Katiyar) [reply server/http.go]: @raghavaggarwal2308 The function call is useful in validating the userID, should we remove it?

Comment (raghavaggarwal2308) [reply server/http.go]: I dont think there is any need to do that, please remove it

**Review by raghavaggarwal2308 — APPROVED** [review]

**Comment by Kshitij-Katiyar** [comment]
Created a PR on mattermost org's repo for this plugin, hence closing this one

**Pull request closed without merging**